src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -86,6 +86,17 @@ internal static class ParenthesizedExpressionSyntaxExtensions
                 return false;
             }
 
+            // Don't remove parentheses around `<` and `>` if there's a reasonable chance that it might
+            // pair with the opposite form, causing them to be reinterpreted as generic syntax. See
+            // https://github.com/dotnet/roslyn/issues/43934 for examples.
+            if (expression.IsKind(SyntaxKind.GreaterThanExpression, SyntaxKind.LessThanExpression) &&
+                nodeParent is ArgumentSyntax)
```

Comment:
This seems to be able to be simplified to just `IsKind`, isn't it?

Reply:
my preference is to not use Kind, as it won't work if we ever have multiple kinds here (which we absolutely could have for things like Arguments).